<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drawingRoom</class>
 <widget class="QWidget" name="drawingRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
      <property name="topMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QWidget" name="sidebar" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sidebartabs" native="true">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(221, 252, 216, 0.5);
border-radius: 30px;</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,12">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_8" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>167793</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgba(221, 252, 216, 1);
border-radius: 30px;

</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="topMargin">
                <number>11</number>
               </property>
               <item>
                <widget class="QPushButton" name="collapse">
                 <property name="minimumSize">
                  <size>
                   <width>40</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="buttonholder" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>160</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">border: 7px solid rgba(174, 243, 163, 1);
border-radius: 20px;
</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>4</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>4</number>
                  </property>
                  <property name="bottomMargin">
                   <number>6</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="pages">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="shapes">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="symbols">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="settings">
                    <property name="minimumSize">
                     <size>
                      <width>40</width>
                      <height>40</height>
                     </size>
                    </property>
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTabWidget" name="sidebartabs_2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="pageTab">
               <attribute name="title">
                <string>Tab 1</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QWidget" name="widget_3" native="true">
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QPushButton" name="create_page">
                     <property name="styleSheet">
                      <string notr="true">color: #000000;
background-color: #FFFFFF;</string>
                     </property>
                     <property name="text">
                      <string>Create Page</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="delete_page">
                     <property name="styleSheet">
                      <string notr="true">color: #000000;
background-color: #FFFFFF;</string>
                     </property>
                     <property name="text">
                      <string>Delete Page</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="previous_page">
                     <property name="styleSheet">
                      <string notr="true">color: #000000;
background-color: #FFFFFF;</string>
                     </property>
                     <property name="text">
                      <string>Previous Page</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="next_page">
                     <property name="styleSheet">
                      <string notr="true">color: #000000;
background-color: #FFFFFF;</string>
                     </property>
                     <property name="text">
                      <string>Next Page</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="shapTabe">
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_6"/>
              </widget>
              <widget class="QWidget" name="symbolTab">
               <attribute name="title">
                <string>Page</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="settingstab">
               <attribute name="title">
                <string>Page</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">border</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="topbar" native="true">
           <property name="minimumSize">
            <size>
             <width>800</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgba(221, 252, 216, 1);

border-radius: 30px;</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0,0,0,0,0,0,0,0">
            <item>
             <widget class="QPushButton" name="leaveRoom">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="name">
              <property name="font">
               <font>
                <pointsize>19</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: #000;</string>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="select">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="article">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="toggle" native="true">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 7px solid rgba(174, 243, 163, 1);
border-radius: 20px;
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QPushButton" name="draw">
                 <property name="styleSheet">
                  <string notr="true">background-color: transparent;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="erase">
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">border-radius: 20px;
background-color: rgba(174, 243, 163, 1);
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>10</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="copyCode">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="code">
                 <property name="styleSheet">
                  <string notr="true">color: rgba(74, 160, 60, 1);
</string>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_5" native="true">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="user2">
                 <property name="styleSheet">
                  <string notr="true">color: rgba(74, 160, 60, 1);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_6" native="true">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QPushButton" name="user3">
                 <property name="styleSheet">
                  <string notr="true">color: rgba(74, 160, 60, 1);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background: rgba(174, 243, 163, 0.7);
border-radius: 20px;
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QPushButton" name="user1">
                 <property name="styleSheet">
                  <string notr="true">color: rgba(74, 160, 60, 1);
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="TabletGraphicsView" name="graphics">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: transparent;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabletGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">tabletgraphicsview.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
